_test/test_sqw_file/test_faccess_sqw_v3.m

```diff
@@ -287,6 +288,44 @@ function test_serialize_deserialise_faccess(obj)
             
             assertEqual(fo,fr);
         end
+        %
+        function test_wrong_file_name_activated(obj)
+            ld = sqw_formats_factory.instance.get_loader(obj.sample_file);
+            sample_obj = ld.get_sqw();
+            
+            test_name = 'test_wrong_file_name_activated_1.sqw';
+            targ_file = fullfile(tmp_dir(),test_name);
+            
+            wrt =sqw_formats_factory.instance.get_pref_access(sample_obj);
+            wrt = wrt.init(sample_obj,targ_file);
+            
+            % test file has been stored with name test_name.
+            wrt.put_sqw();
+            test_name_2 = 'test_wrong_file_name_activated_2.sqw';
+            targ_file_2 = fullfile(tmp_dir(),test_name_2);
+            wrt.delete();
+            clob1 = onCleanup(@()delete(targ_file));

```

Comment:
clearUp object?

Reply:
there are target files 'targ_file' and 'targ_file_2' and cleanups 'clob' and 'clob1'  -- it would be easier to follow if there was consistent and more descriptive naming.